git-standup

```diff
@@ -74,12 +74,14 @@ if [[ ! -d ".git" ]]; then
 
         ## Show the detail only if it is a git repository
         if [[ -d ".git" ]] ; then
-            GITOUT=$(eval ${GIT_LOG_COMMAND})
-
-            ## Only output if there is some activity
-            if [[ ! -z "$GITOUT" ]] ;  then
-                echo "${BOLD}${UNDERLINE}${YELLOW}$DIR${NORMAL}"
-                echo "$GITOUT"
+            if GITOUT=$(eval ${GIT_LOG_COMMAND}); then
+              ## Only output if there is some activity
+              if [[ ! -z "$GITOUT" ]] ;  then
+                  echo "${BOLD}${UNDERLINE}${YELLOW}$DIR${NORMAL}"
+                  echo "$GITOUT"
+              fi
+            else
+              echo "Repository under $DIR could not be queried. Missing initial commit?"
```

Comment:
My thought was that the program could continue to process the repos that had initial commits, but since "set -e" is in the file it will fail anyways, so you're absolutely right. Pushed fix.